### DISCUSSION TITLE
Temario i2

### DISCUSSION THREAD

**Discussion by Osiries1111** [IIC2133-PUC/2024-2#180] [Preguntas Materia] [answered]
Hola!

No me queda claro desde qué clase entra para la I2, pues enviaron muchos correos de corrección respecto al temario, espero que me puedan decir con más exactitud qué entra.

Quedo atenta

**Comment by isabellacherubini** [comment, marked as answer]
Hola! El temario oficial es el mencionado en el último anuncio
1. Tablas de Hash
2. Backtracking
3. Greedy 
4. Programación Dinámica
5. Grafos y DFS

por lo tanto entra desde hash hasta el final del ppt de dfs
cualquier duda me cuentas :)

**Reply by Lexiesm** [reply]
Entonces B+ no entra? me refiero al ppt de la clase 12 (me queda la duda porque en uno de esos anuncios decía como tablas de hash (B+)
de antemano gracias!

**Reply by isabellacherubini** [reply]
Hola! (disculpa la demora), sería solo lo indicado en el temario, es decir, b+ no entra
